sdk/ml/azure-ai-ml/tests/feature_store/e2etests/test_feature_store.py

```diff
@@ -100,9 +101,7 @@ def test_feature_store_create(
         assert create_fs.offline_store.target
         assert create_fs.online_store.target == online_store_target
 
-        fs_poller = client.workspaces.begin_delete(
-            name=fs1.name, delete_dependent_resources=True, permanently_delete=True
-        )
+        fs_poller = client.feature_stores.begin_delete(name=fs1.name, delete_dependent_resources=True)
```

Comment:
why removed the "permanently_delete=true"?

Reply:
feature store delete doesn't have this parameter.